src/main/antlr4/soql/SOQL.g4

```diff
@@ -61,6 +61,10 @@ keyword             : GROUP
                     | CATEGORY
                     ;
 
+selectKey           : DATA
+                    | SCOPE
```

Comment:
We are only fixing the issue for Data and Scope keywords. Do we also want to consider other keywords like end, etc?